### ISSUE TITLE
compiled CLJS code included in the jar

### ISSUE THREAD

**Issue by stask** [victorvoid/formap#3]
Getting following when i add formap to my shadow-cljs project:
```
$ shadow-cljs watch app
shadow-cljs - config: /Users/stask/projects/aktibo/awesomo/awesomo-life/shadow-cljs.edn  cli version: 2.8.40  node: v12.5.0
shadow-cljs - updating dependencies
shadow-cljs - dependencies updated
[2019-07-09 12:46:34.914 - INFO] filename violation for ns reagent.ratom, got: js/release/reagent/ratom.cljs expected: reagent/ratom.cljs (or .cljc)
[2019-07-09 12:46:34.936 - INFO] filename violation for ns clojure.set, got: js/release/clojure/set.cljs expected: clojure/set.cljs (or .cljc)
[2019-07-09 12:46:34.936 - INFO] filename violation for ns reagent.impl.component, got: js/release/reagent/impl/component.cljs expected: reagent/impl/component.cljs (or .cljc)
...
```

This is likely because the formap-0.1.2.jar contains a 'js' folder with compiled cljs files (reagent, corelib).

**Comment by victorvoid** [comment]
Thanks for your report, you can test again ? Try version [formap "0.1.3"]

**Comment by stask** [comment]
Yep, issue fixed in 0.1.3, thanks!
